introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>How about putting notes at the end of the notebook?</p><p> </p><p>Also, how about input format before everything else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>We decided to keep notes next to supported parameters. Moved input format description up.</p>